tests/background/tab-state-test.js

```diff
@@ -134,15 +134,19 @@ describe('TabState', () => {
   describe('#updateAnnotationCount', () => {
     let clock;
     let getStub;
+    let uriForBadgeRequestStub;
     const INITIAL_WAIT_MS = 1000;
     const MAX_WAIT_MS = 3000;
+    const CACHE_EXPIRATION_MS = 3000;
 
     beforeEach(() => {
       clock = sinon.useFakeTimers();
       getStub = sinon.stub();
+      uriForBadgeRequestStub = sinon.stub();
       $imports.$mock({
         './uri-info': {
-          getAnnotationCount: getStub,
+          fetchAnnotationCount: getStub,
```

Comment:
Might be a good time to rename `getStub` to `fetchAnnotationCountStub`.

Reply:
👍🏼